packages/dashboard/src/components/tasks/tasks-app.tsx

```diff
@@ -91,44 +99,54 @@ export const TasksApp = React.memo(
         setWorkcells(Object.keys(resourceManager.dispensers.dispensers));
       }, [resourceManager]);
 
+      const GET_LIMIT = 10;
```

Comment:
Could you add a `TODO` above this line to hopefully parameterize this value in our next set of fixes?

Reply:
`const GET_LIMIT = 10; // TODO PARAMETERIZE THIS VARIABLE`
Something like that?

Reply:
Something above it would work,
```
// TODO: parameterize this variable
const GET_LIMIT = 10;
```